ramalama/quadlet.py

```diff
@@ -77,7 +80,12 @@ def generate(self):
 """
             )
 
-    def gen_volume(self):
+    def gen_chat_template_volume(self):
+        if os.path.exists(self.chat_template):
+            return f"Mount=type=bind,src={self.chat_template},target={MNT_CHAT_TEMPLATE_FILE},ro,Z"
```

Comment:
Use z not Z, since you will probably use this template file for multiple containers.